docs/configuration.md

```diff
@@ -230,24 +250,43 @@ spec:
         - name: cm1
         - name: cm2
           key: file21.txt
+        - name: cm3
+          mountPath: /my/cm3/path
       secrets:
         - name: secret1
           key: file3.txt
+        - name: secret2
+          mountPath: /my/secret2/path
 ```
 
 The Operator will either get all entries from the specified object (if no key is specified) or will pick the specified one, creating volumes per object and mounting the files to the Backstage container.
+Since **v1alpha3 (v0.4)** Backstage CRD introduced **mountPath** field which allows to mount ConfigMap or Secret to specified path. A combination of key/mountPath fields impacts on whether the Volume will be mounted with or without [subPath](https://kubernetes.io/docs/concepts/storage/volumes/#using-subpath) as following:
+* If nothing specified: each key/values will be mounted as filename/content with **subPath**
+* If **key** specified, with or without **mountPath**: the specified key/value will be mounted with **subPath**
+* If only **mountPath** specified: a directory containing all the key/values will be mounted w/o **subPath**
+
+**Note**: A volume mounted with **subPath** is not [automatically updated by Kubernetes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#mounted-configmaps-are-updated-automatically). So, by default, the Operator watches such ConfigMaps/Secrets and refresh Pod in case if it was changed. 
+  
+**Note:** To limit read access to Secrets by the Operator Service Account (for security reasons), we only support mounting files from Secrets if a key is specified.
```

Comment:
Not sure I totally understand, guessing it is about the fact that mountPath without key is valid for secrets. Added comments to example and refine the note, HTH, otherwise please elaborate